src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/name-property/name-property.component.html

```diff
@@ -1,11 +1,11 @@
 <div [formGroup]="group">
```

Comment:
Don't pass prettier in the html

Reply:
I didn't :), some white spaces where removed